docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='SelectList'
+  />
+  <Button
+    accessibilityControls="basic-dropdown-example"
+    accessibilityHaspopup
+    iconEnd="arrow-down"
+    text="Dropdown"
+    inline
+  />
+</Flex>`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      SelectList comes with Label built-in: just use the \`label\` prop. Be sure to provide a unique id so the Label is associated with the correct SelectList.`}
     />
-  );
-}
-    `}
-  />,
+  </MainSection>,
 );
 
 card(
-  <Example
-    id="helperText"
-    name="Example: Helper Text"
-    description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
-    defaultCode={`
-function Example(props) {
-  const [country, setCountry] = React.useState('belgium')
-  const countries = [
-    {
-      value: "belgium",
-      label: "Belgium"
-    },
-    {
-      value: "france",
-      label: "France"
-    },
-    {
-      value: "usa",
-      label: "USA"
-    }
-  ];
-  return (
-    <Box padding={2} color="white">
-      <SelectList
-        id="country"
-        name="country"
-        onChange={({ value }) => setCountry(value)}
-        options={countries}
-        placeholder="Select country"
-        label="Country"
-        helperText="Pick your favourite country (the one with the best fries)"
-        value={country}
+  <MainSection name="Variants">
+    <MainSection.Subsection title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="Large"
+        description={`Use \`lg\` as the default size within Pinterest products.`}
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        size='lg'
+      />`}
       />
-    </Box>
-  );
-}
-`}
-  />,
+      <MainSection.Card
+        cardSize="md"
+        title="Medium"
+        description={`Use \`md\` on denser surfaces, such as Business products or internal tools.`}
+        defaultCode={`<SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Austin', value: 'austin'},
+          {label:'Los Angeles', value: 'la'},
+          {label:'New York', value: 'nyc'},
+          {label:'San Francisco', value: 'sf'},
+          {label:'Seattle', value: 'seattle'},
+        ]}
+        size='md'
+      />`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Helper text"
```

Comment:
I'd find a more Pinterest-relevant example for hypertext. I think we would not add a helper text to select a country.

Reply:
![image](https://user-images.githubusercontent.com/10593890/110534005-95469480-80ec-11eb-8b52-104abde85557.png)

Reply:
This one might be a hard one to find, so maybe more Pinterest-inspired.